index.js

```diff
@@ -8,6 +8,71 @@ const checkWIPModule = require('./lib/checkWipDraftPR');
 
 const whitelistedAccounts = (
   (process.env.WHITELISTED_ACCOUNTS || '').toLowerCase().split(','));
+const checksWhitelist = {
+  'oppia-android': {
+    'opened': ['cla-check'],
+    'reopened': [],
+    'labeled': [],
+    'synchronize': [],
+    'closed': [],
+    'edited': []
+  },
+  'oppia': {
+    'opened': [
+      'cla-check',

```

Comment:
Similarly, I suggest moving each of these checks to their own constant (and referencing them below in the switch statement) so that you can move comments like the one below ('Prevent user from reopening the PR') to be next to the constant itself. If each constant has context, that'll make it easier for repositories to decide which functionality they want to enable & why without having to dig into each of these checks to understand what functionality that they provide.

Reply:
Done.